test/e2e/scheduling/priorities.go

```diff
@@ -582,13 +603,15 @@ func createRC(ns, rsName string, replicas int32, rcPodLabels map[string]string,
 	return rc
 }
 
-func addRandomTaintToNode(cs clientset.Interface, nodeName string) *v1.Taint {
-	testTaint := v1.Taint{
-		Key:    fmt.Sprintf("kubernetes.io/e2e-taint-key-%s", string(uuid.NewUUID())),
+func getRandomTaint() v1.Taint {
+	return v1.Taint{
+		Key:    fmt.Sprintf("kubernetes.io/scheduling-priorities-e2e-taint-key-%s", string(uuid.NewUUID())),
```

Comment:
Is there a max taint key length we should be watching out for?  Labels are capped at 63; the hardcoded string part here is 52 chars and `string(uuid.NewUUID())` appears to be 37 chars

Reply:
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#taint

>The key must begin with a letter or number, and may contain letters, numbers, hyphens, dots, and underscores, up to 253 characters.

ok, this is well within the limit

Reply:
@spiffxp did that limit recently increase? Trying to port this to openshift I am seeing an error that this key is too long: `name part must be no more than 63 characters`

Reply:
@damemi this is failing in the upstream CI too, due to this key limitation
https://testgrid.k8s.io/sig-network-gce#gci-gce-serial&width=5
```
                       Type: "FieldValueInvalid",
                        Message: "Invalid value: \"kubernetes.io/scheduling-priorities-e2e-taint-key-c19ab4b9-07b1-4a86-9a0f-52c1a6028392\": name part must be no more than 63 characters",
                        Field: "metadata.taints[0].key",
                    },
                ],
```

Reply:
ping @spiffxp ^^^

Reply:
filed https://github.com/kubernetes/kubernetes/pull/99020

we may also need to update the docs.